COMPATIBILITY.md

```diff
@@ -0,0 +1,40 @@
+# Instrumented Library Compatibility
+
+Instrumentation is provided for the following Go packages.
+
+- [database/sql](#database/sql)
+- [github.com/segmentio/kafka-go](#github.com/segmentio/kafka-go)
+- [google.golang.org/grpc](#google.golang.org/grpc)
+- [net/http](#net/http)
+
+## database/sql
+
+[Package documentation](https://pkg.go.dev/database/sql)
+
+Supported version ranges: 
+
+- 1.12.0 to 1.22.5
+
+## github.com/segmentio/kafka-go
+
+[Package documentation](https://pkg.go.dev/github.com/segmentio/kafka-go)
+
+Supported version ranges: 
+
+- v0.4.1 to v0.4.47
+
+## google.golang.org/grpc
+
+[Package documentation](https://pkg.go.dev/google.golang.org/grpc)
+
+Supported version ranges: 
+
+- v1.14.0 to v1.66.0
+
+## net/http
+
+[Package documentation](https://pkg.go.dev/net/http)
+
+Supported version ranges: 
+
+- 1.12. to 1.22.5
```

Comment:
I propose to use the same naming as in https://pkg.go.dev/net/http?tab=versions:

```suggestion
- `go1.12 to `go1.22.5`
```